src/generate-note-controller.js

```diff
@@ -184,6 +184,7 @@ module.exports = function(scribe){
         }
         var selectionIsCollapsed = (markers.length === 1);
 
+        /*
```

Comment:
That's fine, what I meant is that it's good practice to document commented out code to make it clear (1) why it was commented out (to fix a bug? to address a functionality change?) and (2) when it can be retired (once a decision has been made? once an external PR has been merged?).